app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,18 @@
+// Greeting.js
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getGreeting } from '../redux/greeting/greetingSlice';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const { greeting } = useSelector((state) => state.greeting);
+
+  useEffect(() => {
+    dispatch(getGreeting());
+  }, []);
+
+  
+  return <div>{greeting.greeting}</div>;
+};
```

Comment:
- [ ] Nice work using `redux` to get the greeting from the back-end server :clap: However there are some error when rendering your `Greeting` component. It sames like you're getting this error because the the **redux store** is not available in the Greeting component. 
  This can be fixed by wrapping your `App` component in the `application.js` in `Provider` from the `react-redux` package.